extensions/airsy/src/index.tsx

```diff
@@ -0,0 +1,18 @@
+import { List } from "@raycast/api";
+import useStations from "./services/GIOS";
+import StationDetails from "./components/StationDetails";
+import { IStationResult } from "./types";
+
+export default function Command() {
+  const { stations, isLoading } = useStations();
+  return (
+    <List navigationTitle={`Airsy - AllStations`} isLoading={isLoading} isShowingDetail>
```

Comment:
```suggestion
    <List isLoading={isLoading} isShowingDetail>
```
Don't change navigationTitle in the root command